### ISSUE TITLE
Would this ever be available for Windows?

### ISSUE THREAD

**Issue by skiletro** [ccMSC/ckb#555]
I prefer this over the regular CUE software. Would that be possible?

**Comment by chillgamingAZ** [comment]
it shouldn't be too difficult to compile a windows version

**Comment by charendoff** [comment]
just get the actual corsair software

**Comment by skiletro** [comment]
I do. I dualboot Windows 10 and Linux Mint. CUE is way too buggy and complicated for my liking. I just prefer ckb over cue.

**Comment by light2yellow** [comment]
No, this will never ever be available for Windows. It has very different device handling and this requires a new project to be started. Which is rather useless because this project was intended to provide Windows' features on other popular platforms.
